src/operator/numpy/random/np_uniform_op.h

```diff
@@ -0,0 +1,211 @@
+                         const std::vector<TBlob> &outputs) {
+  using namespace mshadow;
+  using namespace mxnet_op;
+  const NumpyUniformParam &param = nnvm::get<NumpyUniformParam>(attrs.parsed);
+  CHECK_EQ(outputs.size(), 1);
+  Stream<xpu> *s = ctx.get_stream<xpu>();
+
+  // Generate base random number.
+  Random<xpu, float> *prnd = ctx.requested[0].get_random<xpu, float>(s);
+  Tensor<xpu, 1, float> uniform_tensor =
+      ctx.requested[1].get_space_typed<xpu, 1, float>(Shape1(outputs[0].Size()),
+                                                      s);
+  prnd->SampleUniform(&uniform_tensor, 0, 1);
+  mxnet::TShape new_lshape, new_hshape, new_oshape;
+
+  // [scalar scalar] case
+  if (inputs.size() == 0U) {
+    MSHADOW_TYPE_SWITCH(outputs[0].type_flag_, OType, {
+      mxnet_op::Kernel<uniform_two_scalar_kernel<OType>, xpu>::Launch(
+          s, outputs[0].Size(), param.low.value(), param.high.value(),
+          uniform_tensor.dptr_, outputs[0].dptr<OType>());
+    });
+  } else if (inputs.size() == 1U) {
+    // [scalar tensor], [tensor scalar] case
+    int ndim = FillShape(inputs[0].shape_, inputs[0].shape_, outputs[0].shape_,
+                         &new_lshape, &new_lshape, &new_oshape);
+    int scalar_pos;
+    float scalar_value;
+    // int type_flag = param.t;
+    if (param.low.has_value()) {
+      scalar_pos = 0;
+      scalar_value = param.low.value();
+    } else {
+      scalar_pos = 1;
+      scalar_value = param.high.value();
+    }
+    MSHADOW_TYPE_SWITCH(inputs[0].type_flag_, IType, {
+      MSHADOW_TYPE_SWITCH(outputs[0].type_flag_, OType, {
+        BROADCAST_NDIM_SWITCH(ndim, NDim, {
+          mshadow::Shape<NDim> oshape = new_oshape.get<NDim>();
+          mshadow::Shape<NDim> stride =
+              mxnet_op::calc_stride(new_lshape.get<NDim>());
+          mxnet_op::Kernel<uniform_one_scalar_kernel<NDim, IType, OType>,
+                           xpu>::Launch(s, outputs[0].Size(), scalar_pos,
+                                        stride, oshape, inputs[0].dptr<IType>(),
+                                        scalar_value, uniform_tensor.dptr_,
+                                        outputs[0].dptr<OType>());
+        });
+      });
+    });
+  } else if (inputs.size() == 2U) {
+    // [tensor tensor] case
+    int ndim = FillShape(inputs[0].shape_, inputs[1].shape_, outputs[0].shape_,
+                         &new_lshape, &new_hshape, &new_oshape);
+    MSHADOW_TYPE_SWITCH(inputs[0].type_flag_, IType, {
+      MSHADOW_TYPE_SWITCH(outputs[0].type_flag_, OType, {
+        BROADCAST_NDIM_SWITCH(ndim, NDim, {
+          mshadow::Shape<NDim> oshape = new_oshape.get<NDim>();
+          mshadow::Shape<NDim> lstride =
+              mxnet_op::calc_stride(new_lshape.get<NDim>());
+          mshadow::Shape<NDim> hstride =
+              mxnet_op::calc_stride(new_hshape.get<NDim>());
+          mxnet_op::Kernel<uniform_kernel<NDim, IType, OType>, xpu>::Launch(
```

Comment:
Get rid of all `mshadow::` and `mxnet_op::` since you already did `using namespace mshadow;` and `using namespace mxnet_op;`

Reply:
Done, thx for advice